tokens/custom-spectrum/custom-large-vars.css

```diff
@@ -120,4 +120,14 @@ governing permissions and limitations under the License.
 	--spectrum-assetcard-content-font-size: var(--spectrum-body-size-xs);
 
   --spectrum-tooltip-animation-distance: 5px;
+
+  --spectrum-picker-side-label-quiet-padding-block-start-small: -1px;
+  --spectrum-picker-side-label-quiet-padding-block-start-medium: -1px;
+  --spectrum-picker-side-label-quiet-padding-block-start-large: -1px;
+  --spectrum-picker-side-label-quiet-padding-block-start-xlarge: 0px;
+
+  --spectrum-fieldlabel-picker-side-label-quiet-padding-block-start-small: 5px;
+  --spectrum-fieldlabel-picker-side-label-quiet-padding-block-start-medium: 11px;
+  --spectrum-fieldlabel-picker-side-label-quiet-padding-block-start-large: 14px;
+  --spectrum-fieldlabel-picker-side-label-quiet-padding-block-start-xlarge: 18px;
```

Comment:
Are these values coming from the design team? The numbers seem a bit magical ✨

Reply:
these are a big magical, This is adjusting each of these, the quiet picker and quiet field label, to be inline with adjacent components (this case this switch). I am awaiting designs thoughts on these values and if they should be tokens as well.

Reply:
I'm also a little curious how these look on retina vs non-retina displays. Will things still be aligned?

Reply:
I did look into this, the macbook I work on does have a built in retina display while my external monitor is an IPS display. 

On the retina monitor (red underline) there is about a half pixel difference 

<img width="820" alt="Screenshot 2023-12-11 at 10 46 11 AM" src="https://github.com/adobe/spectrum-css/assets/63808889/80f339ac-5ca4-4cf2-96e8-8f68a6e02b5e">